client/post-editor/editor-ground-control/index.jsx

```diff
@@ -370,18 +291,14 @@ const EditorGroundControl = React.createClass( {
 						{ this.getPreviewLabel() }
 					</button>
 					<div className="editor-ground-control__publish-combo">
-						<button
-							className="editor-ground-control__publish-button button is-primary"
-							onClick={ this.onPrimaryButtonClick }
-							disabled={ ! this.isPrimaryButtonEnabled() }
+						<EditorPublishButton
+							{ ...this.props }
 							tabIndex={ 5 }
-						>
-							{ this.getPrimaryButtonLabel() }
-						</button>
-						{ siteUtils.userCan( 'publish_posts', this.props.site ) &&
+						/>
+						{ this.canSchedulePost() &&
 							<button
 								ref="schedulePost"
-								className="editor-ground-control__time-button button is-primary"
```

Comment:
Since you're hiding this on smaller viewports, why not just keep the `is-primary` class on the button element, rather than applying the mixin in the style selector?